### ISSUE TITLE
When favicon not set, public view generates empty <link> element.

### ISSUE THREAD

**Issue by kimisgold** [omeka/omeka-s#2268]
In the public views, if a site admin has not set a favicon, the view generates an empty `<link rel="icon">`, which parsing tools have tripped over. My understanding was that the fallback should be an Omeka favicon, or none at all.

(Initially reported in https://github.com/omeka-s-themes/cozy/issues/42#issuecomment-2699474422.)
